src/App.js

```diff
@@ -1,23 +1,42 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { Clock } from './components/CLock';
 
 import './App.scss';
 
 const App = () => {
-  setInterval(() => {
-    const date = new Date();
-
-    // eslint-disable-next-line
-    console.log(date.toLocaleTimeString());
-  }, 1000);
+  const [isClockVisible, setClockVisibility] = useState(true);
+  const [clockName, setClockName] = useState(Math.floor(Math.random() * 100));
 
   return (
     <div className="App">
-      <h1>React clock</h1>
-      <p>
-        Current time:
-        {' '}
-        {/* Print the time here instead of DevTools */}
-      </p>
+      <div className="App__time">
+        {
+          isClockVisible ? <Clock name={clockName} /> : ''
```

Comment:
Це можна так скоротити
```suggestion
          isClockVisible && <Clock name={clockName} />
```